roles/ceph-mds/tasks/docker/pre_requisite.yml

```diff
@@ -19,6 +19,11 @@
     state: present
   when: ansible_os_family == 'RedHat'
 
+- name: install six
```

Comment:
Unfortunately, I need to ensure that a certain version of six is installed, so I can't use the with_items loop.  I did add a comment to indicate this.